modules/web/js/ballerina/components/editable-text.jsx

```diff
@@ -26,33 +26,46 @@ class EditableText extends React.Component {
             editing: false
         };
 
+        const textBoxPadding = 3;
```

Comment:
IMO this should be captured in CSS, if we make the default too granular it will be difficult to maintain.